wremnants/plot_tools.py

```diff
@@ -74,38 +74,46 @@ def figure(href, xlabel, ylabel, ylim=None, xlim=None,
 def figureWithRatio(href, xlabel, ylabel, ylim, rlabel, rrange, xlim=None,
     grid_on_main_plot = False, grid_on_ratio_plot = False, plot_title = None, title_padding = 0,
     x_ticks_ndp = None, bin_density = 300, cms_label = None, logy=False, logx=False,
-    width_scale=1, automatic_scale=True
+    width_scale=1, automatic_scale=True, only_ratio=False
 ):
     fig, xlim = cfgFigure(href, xlim, bin_density, width_scale, automatic_scale)
     
-    ax1 = fig.add_subplot(4, 1, (1, 3)) 
+    if not only_ratio:
+        ax1 = fig.add_subplot(4, 1, (1, 3)) 
+        ax1.set_xlabel(" ")
+        ax1.set_ylabel(ylabel)
+        ax1.set_xlim(xlim)
+
+        if ylim:
+            ax1.set_ylim(ylim)
+        else:
+            ax1.autoscale(axis='y')
+
+        if logy:
+            ax1.set_yscale('log')
+        if grid_on_main_plot:  ax1.grid(which = "both")
+        if plot_title: ax1.set_title(plot_title, pad = title_padding)
```

Comment:
The prefit and postfit plots in the CI are missing the "CMS Work in Progress" title and luminosity. I suspect it is related to the changes in the order done here, maybe putting the "hep.cms.text(cms_label)" before this block will fix it?

Reply:
It's because I removed it and didn't readd it, my bad